### ISSUE TITLE
Dropdown menus instantly closing

### ISSUE THREAD

**Issue by brittyazel** [flathub/us.zoom.Zoom#391]
I'm having an issue since the latest update where drop down menu's in Zoom will instantly close. I can see the menu flash for a split second, but then it disappears.

I.e. when I click my icon in the upper right corner of the main Zoom window, the menu flashes and closes in a about 1 frame, meaning that it's impossible to click any menu options. This is also true for within a call, when trying to change the view type from gallery to presenter, etc.

I don't see any console out put indicating an issue, so I'm not sure what's going on.

**Comment by real-or-random** [comment]
Same here, GNOME on Wayland.

**Comment by kevenwyld** [comment]
I'm also experiencing this on sway.  You can still join calls and it's mostly functional, but you can't do anything that requires a drop-down menu.  They briefly flash on the screen and then disappear when clicked.

**Comment by Lcchy** [comment]
Yes its the same for me on Sway too, using the flatpak or the native (rpm) package.

**Comment by naythanN** [comment]
Also have this issue under hyprland

**Comment by yotam-medini** [comment]
Same here with firefox on
$ cat /etc/lsb-release 
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=21
DISTRIB_CODENAME=vanessa
DISTRIB_DESCRIPTION="Linux Mint 21 Vanessa"

**Comment by brittyazel** [comment]
As an update to this, I no longer have this issue. This is on Arch Linux updated as of Jun 21st, 2024, with GNOME 45.2, running Zoom from Flathub version 6.0.12. Something seems to have stabilized the situation either on the packaging end and/or the Zoom client itself, but things are pretty good at the moment.
